packages/eslint-config/overrides/index.js

```diff
@@ -0,0 +1,7 @@
+// TODO: shareable config 里面暂时无法使用 extends ， 所以交给项目配置
+module.exports = {
```

Comment:
```suggestion
module.exports = {
  typescript: require('./typescript'),
  babel: require('./babel'),
  jest: require('./jest'),
  react: require('./react'),
  overrides: [require('./babel'), require('./jest'), require('./react'), require('./typescript'),]
};
```

Is that better or not?

Reply:
maybe better, but I like to import clearer.

Reply:
Shall we add `.mdx` related rules? Which can handle `.md/.mdx` files.